rbac/management/migrations/0020_add_unique_to_display_name.py

```diff
@@ -0,0 +1,31 @@
+# Generated by Django 2.2.4 on 2020-08-04 14:42
+
+from django.db import migrations, models
+
+
+def update_display_name(apps, schema_editor):
+    # get role model
+    Role = apps.get_model("management", "Role")
+
+    # iterate through all Access
+    for role in Role.objects.all():
+        # get the permission context from the permission
+        role.display_name = role.name
+        role.save()
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("management", "0019_recreate_display_name_field"),
+    ]
+
+    operations = [
+        migrations.AlterField(
```

Comment:
Curious, what would happen if we remove this line?

Reply:
I'll test this, I think this was originally here to allow for the data migration to run with a default value, but I'll test just doing the data migration then adding on the uniqueness and default constraints afterwards.